src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -282,12 +289,43 @@ export default App.extend({
     const owner = Radio.request('entities', 'clinicians:model', filters.clinicianId);
     const team = Radio.request('entities', 'teams:model', filters.teamId);
 
-    this.showChildView('title', new ListTitleView({
+    const listTitleView = this.showChildView('title', new ListTitleView({
       owner,
       team,
       worklistId: this.worklistId,
       isFlowList: this.getState().isFlowType(),
+      showOwnerDroplist: this.canViewAssignedActions,
```

Comment:
The clinician filter is only shown if the user has the `'app:worklist:clinician_filter'` permission.